lib/plugins/aws/deployFunction.js

```diff
@@ -22,6 +22,8 @@ class AwsDeployFunction {
       path.join(this.serverless.serviceDir || '.', '.serverless');
     this.provider = this.serverless.getProvider('aws');
 
+    this._shouldEnsureFunctionState = false;
```

Comment:
I believe we can simplify it this way 👍

Reply:
Sorry, I've just noticed my push didn't go through - we need to record it both for `updateFunctionCode` and `updateFunctionConfiguration` because configuration update can be skipped and function might be still in bad state after only code update - I've changed the variable a little bit